Description: Adapt plugin building process
 Proceed with following changes:
   * don't build embedded plugins in moment when uWSGI binary is builded. Any
     of these plugins is builded separately for it's own package
   * place builded plugin in current directory instead of UWSGI_PLUGIN_DIR
     (as UWSGI_PLUGIN_DIR is redefined in another patch to directory in
     /usr/lib hierarchy)
Author: Leonid Borisenko <leo.borisenko@gmail.com>
Forwarded: not-needed
Last-Update: 2011-01-18

Index: uwsgi/uwsgiconfig.py
===================================================================
--- uwsgi.orig/uwsgiconfig.py	2011-01-31 21:35:15.000000000 +0200
+++ uwsgi/uwsgiconfig.py	2011-01-31 21:35:23.000000000 +0200
@@ -141,12 +141,6 @@
 		if ret != 0:
 			sys.exit(1)
 
-	if len(PLUGINS) > 0:
-		print("*** uWSGI building plugins ***")
-		for plugin in PLUGINS:
-			print("*** building plugin: %s ***" % plugin)
-			build_plugin("plugins/%s" % plugin)
-
 	print("*** uWSGI linking ***")
 	ldline = "%s -o %s %s %s %s" % (GCC, bin_name, ' '.join(ldflags), ' '.join(map(add_o, gcc_list)), ' '.join(libs))
 	print(ldline)
@@ -467,7 +461,7 @@
 	p_cflags.insert(0, '-I.')
 
 	plugin_base = path + '/' + up.NAME + '_plugin'
-	plugin_dest = UWSGI_PLUGIN_DIR + '/' + up.NAME + '_plugin'
+	plugin_dest = './' + up.NAME + '_plugin'
 
 	shared_flag = '-shared'
 
